docroot/themes/humsci/humsci_basic/src/scss/partials/components/_structured-card.scss

```diff
@@ -46,6 +46,8 @@
     flex-wrap: wrap;
 
     .views-field {
+      display: flex;
```

Comment:
Not a fan of this (and the `display: block` below), we're forcing styles that could bring problems in certain views just for removing a div. It might be better to revert to the original state.